rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -47,6 +48,7 @@ private BookKeeperServer()
     public static void main(String[] args)
     {
         conf = new Configuration();
+        conf.addResource(new YarnConfiguration());
```

Comment:
What will happen if there is no yarn ? I know thats not possible in Qubole or EMR. Will Rubix fail to start up ? Just for my information. I think thats fine as a start.